README.md

```diff
@@ -138,6 +147,15 @@ linkStyle default opacity:0.5
   preferences_controller --> base_controller;
   preferences_controller --> controller_utils;
   preferences_controller --> keyring_controller;
+  profile-sync-controller --> base_controller;
+  profile-sync-controller --> keyring-api;
```

Comment:
This graph just shows interdependencies between monorepo packages. Did you run `yarn update-readme-content` to generate this list? It should fill in the package names for you.

Reply:
TY for the heads up, I believe a generate package script command was used (I'm not sure off the top of my head). Will run the above script to ensure the correct dependency list.